src/components/Disclaimer/index.tsx

```diff
@@ -8,143 +7,188 @@ export interface DisclaimerProps {
+                - In no event shall the initiator, programmer, auditor, auctioneer, participant or any other person
+                or entity which was/is involved in the creation of, maintenance of or any other form of interaction
+                or support of the DutchX be liable for any indirect, special, incidental, consequential,
+                or exemplary damages of any kind arising out of or in any way related to the use of the DutchX,
+regardless of the form of action, whether based in contract, tort or any other legal or equitable theory.<br />
 
-- You will indemnify, defend, and hold harmless the initiator, programmer, auditor, auctioneer, participant
-or any other person or entity which is involved in the code maintenance
-or other form of interaction with the DutchX from and against all claims, demands, actions, damages, losses,
-costs, and expenses that arise from or relate to your responsibilities
-under this Disclaimer or your violation of this Disclaimer. <br/>
+                - You will indemnify, defend, and hold harmless the initiator, programmer,
+                auditor, auctioneer, participant
+                or any other person or entity which is involved in the code maintenance
+                or other form of interaction with the DutchX from and against all claims,
+                demands, actions, damages, losses,
+                costs, and expenses that arise from or relate to your responsibilities
+under this Disclaimer or your violation of this Disclaimer. <br />
 
-- In case that your jurisdiction does not allow the limitation or exclusion of liability for incidental
-or consequential damages you are prohibited to interact with the DutchX.<br/>
+                - In case that your jurisdiction does not allow the limitation or exclusion of liability for incidental
+or consequential damages you are prohibited to interact with the DutchX.<br />
 
-- You recognize and agree on using the DutchX on your own risk. <br/>
+                - You recognize and agree on using the DutchX on your own risk. <br />
 
-- You are legally allowed to buy/sell this kind of (new) asset class. <br/>
-</p>
-      </div>
+                - You are legally allowed to buy/sell this kind of (new) asset class. <br />
+              </p>
+            </div>
 
 
-      <div className="disclaimerBox md-checkbox">
-      <input id="disclaimer5" type="checkbox" required/>
-      <label htmlFor="disclaimer5">
-        <b>
-          The above constitutes a summary of the critical points but is not limited to it.
+            <div className="disclaimerBox md-checkbox">
+              <input id="disclaimer5" type="checkbox" required />
+              <label htmlFor="disclaimer5">
+                <b>
+                  The above constitutes a summary of the critical points but is not limited to it.
           You confirm that you have read the full disclaimer <a href="">available here</a>.
```

Comment:
TODO here so we dont forget

Reply:
Actyally that's a `href` to the current page, not a _to-be-added_ href

Reply:
If it for some reason doesn't lead to `./diclaimer` for you, then it's a bug